website/docs/components/form/file-input/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,31 @@
+## Usage
+
+### When to use
+
+- Use a file input to let users quickly select files from their computer.
+
+### Disabled State
+
+The File Input component allows users to upload documents and images. Disabling it can make users feel confused, frustrated, and unsure about why it’s not working.
```

Comment:
[Suggestion] might recommend against "it's not working"  and instead reference the component and specific state.

```suggestion
The File Input component allows users to upload documents and images. Disabling it can make users feel confused, frustrated, and unsure about why the component is disabled.
```